src/components/ArticleComponents/EditLink/__tests__/__snapshots__/edit-link.test.tsx.snap

```diff
@@ -7,6 +7,8 @@ exports[`EditLink component renders correctly 1`] = `
   >
     <a
       href="https://github.com/nodejs/nodejs.dev/edit/main/content/learn/0002-node-history/index.md"
+      rel="noopener noreferrer"
```

Comment:
Can we create a common component for `External` links with target and rel 🤔 ?

**Ref:** https://github.com/shanpriyan/pokedex/blob/main/src/components/ExternalLink/ExternalLink.jsx

Reply:
Why not. But what does it change concretely?

Reply:
it will be easier to add external links in the future, just href prop is enough instead of adding rel and blank every time. 
Just a react thing :)

Reply:
The thing is that you have to import each time the component that seems to me a little complicated

Reply:
I think for now, this is good enough :)